includes/tax-functions.php

```diff
@@ -236,7 +251,12 @@ function edd_get_tax_rate( $country = '', $region = '', $fallback = true ) {
 	 * @param string $city           City.
 	 * @param string $zip            ZIP code.
 	 */
-	return apply_filters( 'edd_tax_rate', $rate, $country, $region, $address_line_1, $address_line_2, $city, $zip );
+	$rate = apply_filters( 'edd_tax_rate', $rate, $country, $region, $address_line_1, $address_line_2, $city, $zip );
```

Comment:
I was asking this above but then had a second question about this filter so I'll put the two together. I'm curious as to why the address 1/2, city, and zip are being added to this filter, but they're only populated if there is `$_POST` data.

It would seem like in 3.0, adding the `$tax_rate` object as a filter parameter might be useful here? (I know a lot of this was already set and isn't part of this PR but I have questions.)

Reply:
I guess it's meant to be providing the full address used to determine the tax rate. :shrug: As you said, this part isn't new to the PR so I can only guess as much as you lol.

I'm fine with adjusting that but I think it should be a separate PR.